src/pages/position/view/details.jsx

```diff
@@ -0,0 +1,131 @@
+import { Crew } from "@phoenixlan/phoenix.js";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { CardContainer, CardContainerIcon, CardContainerInnerIcon, CardContainerInnerText, CardContainerText, InnerContainer, InnerContainerRow, InnerContainerTitle, InputLabel } from "../../../components/dashboard"
+import { faCode, faGear, faLink, faUserGroup, faUserTie } from "@fortawesome/free-solid-svg-icons"
+import { useEffect, useState } from "react";
+import { PageLoading } from "../../../components/pageLoading";
+
+export const PositionDetails = ({position}) => {
+
+    const [ loading, setLoading ] = useState(true);
+    const [ crew, setCrew ] = useState(undefined);
+    const [ team, setTeam ] = useState(undefined);
+
+    useEffect(async () => {
+        let crew;
```

Comment:
```
let crew = position.crew_uuid ? await Crew.getCrew(position.crew_uuid) : null;
```

This is a major nitpick but I think ternaries are a bit easier to read, and also with this code the value is `null` when a position doesn't exist - not undefined.

There is a semantic difference that doesn't really matter here but its nice to be exact.